src/components/Tabs.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+
+  const selectedTab = tabs.find((tab) => tab.id === tabId);
+
+  return (
+    <>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              data-cy="Tab"
+              key={tab.id}
+              className={classNames(
+                { 'is-active': tab.id === tabId },
+              )}
+            >
+              <Link
+                to={`/tabs/${tab.id}`}
+              >
+                {tab.title}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="block" data-cy="TabContent">
+        {selectedTab
+          ? (
+            `${selectedTab.content}`
+          )
+          : (
+            'Please select a tab'
+          )}
```

Comment:
```suggestion
        {selectedTab
          ? `${selectedTab.content}`
          : 'Please select a tab'}
```

Reply:
Sorry about that now it is everything updated!